linkerd/stack/tracing/src/lib.rs

```diff
@@ -88,16 +88,20 @@ where
     type Future = Instrument<M::Future>;
 
     fn poll_ready(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
-        trace!("poll_ready");
         let ready = self.make.poll_ready(cx);
-        trace!(ready = ready.is_ready());
+        match ready {
+            Poll::Pending => trace!(ready = false, "make"),
+            Poll::Ready(ref res) => trace!(ready = true, ok = res.is_ok(), "make"),
```

Comment:
TIOLI: if `res` is an error, should we include the error in the event?

Reply:
adding an error constraint means adding more type constraints in more places, so would prefer to avoid that for now. we tend to log errors otherwise.,

Reply:
fair, it's not a blocker